stdlib/OpenBLAS_jll/Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "OpenBLAS_jll"
 uuid = "4536629a-c528-5b80-bd46-f80d51c5b363"
-version = "0.3.23+2"
+version = "0.3.23+3"
```

Comment:
I believe you want the +4 build, not the +3 one: https://github.com/JuliaRegistries/General/pull/99646

Reply:
Oh, right! I was wondering what established the patch number...